e2e/cypress/test_module/cypress/support/step_definitions/home-page-general.steps.ts

```diff
@@ -141,3 +157,28 @@ Then("I should see {int} skeletons loading components", (sceletonsCount) => {
     });
   });
 });
+
+When(
+  "I look throw Shop By Category section and click on {string} category",
+
+  (category: string) => {
+    let linkIndex: number;
+
+    switch (category) {
+      case "computer":
+        linkIndex = 0;
+        break;
+      case "tablet":
+        linkIndex = 1;
+        break;
+      case "mobile":
+        linkIndex = 2;
+        break;
+    }
```

Comment:
You can use array instead and access by index

Reply:
fixed